[FEATURE] Créer le repository permettant de sauvegarder les nouveaux format de learner (Pix-11615)

xav-car opened the pull request:
## :unicorn: Problème
Dans la continuité de l'épix de la reforme des imports , on souhaite sauvegarder les infos des learners issus de l'import

## :robot: Proposition
Ajouter 2 méthodes un pour sauvegarder en bdd et l'autre pour désactiver les learners d'une organization

## :rainbow: Remarques

## :100: Pour tester
- On ne peut pas tester pour l'instant, donc vérifier que les tests de l'api passent.

mcampourcy committed:
:recycle: api: Refacto getNextChallengeForCertification to split it for v2/v3

mcampourcy committed:
:sparkles: api: Check if user lang is available for v3 certification

yaf committed:
:recycle: refactor: move organization repo to src dir

Commit:
:recycle: api: asyncify usecase

Commit:
:sparkles: api: add certification version repository

xav-car committed:
chore(orga): update ember-data to 4.12.5

xav-car committed:
fix(orga): fix test afeter update ember-data

Commit:
:recycle: api: reorder imports due to linter configuration modification

This renaming allows us to take into account the fact BOTH V1 and V2 certifications
share the same process of challenge selection

Jeyffrey committed:
feat(api): add TargetProfileForAdmin hasLinkedAutonomousCourse property

Jeyffrey committed:
feat(admin): add linked with autonomous-course info in target-profile details

renovate[bot] committed:
[BUMP] Update dependency @1024pix/stylelint-config to ^5.1.2 (admin)

Jeyffrey committed:
fix(mon-pix): prevent extra assessment checkpoint before final checkpoint

renovate[bot] committed:
[BUMP] Update dependency @1024pix/stylelint-config to ^5.1.2 (mon-pix)

Commit:
tech(api): create onde import seeds

cberthou committed:
:sparkles: admin: add ui to create scoring configurations

Commit:
:recycle: api: rename competence-for-scoring-repository

Renaming to scoring-configuration-repository as it will contain
both general scoring and competence level related methods

Commit:
:card_file_box: api: create scoring-configurations table

Commit:
:sparkles: api: add scoring configuration DB builder

Commit:
:seedling: api: add scoring configuration seed

cberthou committed:
:recycle: api: extracted v3 certification logic into a specific domain object

Commit:
:recycle: api: rename listByLocale to getLatestByDateAndLocale

Commit:
:sparkles: api: add save method for scoring configuration

Commit:
:recycle: api: rename save method in scoring-configuration-repository

This renaming allows us to add the save mmethod for certification scoring configuration
without confusion

Commit:
:recycle: api: rename competences-for-scoring route and contoller to scoring-configuration

This renaming will allow us to gather both certification scoring and competences scoring
related routes

Commit:
:sparkles: api: create route for adding a certification scoring config

Commit:
:sparkles: api: use real certification scoring conf instead of hard coded one

renovate[bot] committed:
[BUMP] Update dependency @1024pix/stylelint-config to ^5.1.3 (orga)

AndreiaPena committed:
✨ certif: use PixRadioButton in new certification candidate modal

Commit:
:sparkles: api: use minimum and maximum mesh level values instead of hardcoded values

Using hardcoded minimum and maximum values forces us to change the minimum and maximum values
provided by the data team on configuration importation as the candidate level must be part of an interval
in order to be scored

igorissen committed:
feat(orga): handle unavailable languages

by defaulting to English

igorissen committed:
feat(certif): handle unavailable languages

by defaulting to English international

AndreiaPena committed:
✨ certif: use PixInput in new certification candidate modal

matthieu-octo committed:
:sparkles: api: add keys for comments translation

Alexandre-Monney committed:
refactor(api): campaign participation routes migration to src

dlahaye committed:
fix(mon-pix): do not display empty grains

Co-authored-by: miranda.lin-guignard <miranda.lin-guignard@pix.fr>

er-lim committed:
docs(api): add missing POLE_EMPLOI_TOKEN_URL description in sample.env

er-lim committed:
docs(api): align SSO section of docker sample.env and API sample.env

er-lim committed:
refactor(api): remove use of AUTHENTICATION_URL and userInfoUrl

matthieu-octo committed:
:sparkles: api: adding a method for recording lower level comment key in the assessment result

Commit:
:recycle: api: use object instead of regular params in scoring repository

er-lim committed:
refactor(mon-pix): remove use of should in footer integration test

matthieu-octo committed:
:sparkles: api: update the assessment result if the lower level complementary certification is acquired during scoring

er-lim committed:
refactor(mon-pix): reorganize footer integration test

mirawlin committed:
feat(mon-pix): dynamically change the background colour of activity and lesson

Co-authored-by: Dimitri Lahaye <dimitri.lahaye@pix.fr>

Commit:
:recycle: api: alter route for saving competence scoring config

er-lim committed:
feat(mon-pix): hide student data protection policy link

in international domain

Commit:
:white_check_mark: api: add missing given-when-then

igorissen committed:
feat(orga): add Banner::LanguageAvailability component

igorissen committed:
feat(orga): set session data attributes

localeNotSupported and localeNotSupportedBannerClosed

igorissen committed:
feat(orga): add method updateDataAttribute

to session service

machestla committed:
tech(orga): replace contains by testing library in participation-filter integration test

igorissen committed:
feat(orga): handle language availability close action

Osirisxxl committed:
🔨 api: sco division certification result csv holds the last certification session joined by a student

Signed-off-by: Jérémie Picard <jeremie.picard@pix.fr>

dlahaye committed:
feat(api): add grain type tag

Co-authored-by: miranda.lin-guignard <miranda.lin-guignard@pix.fr>

igorissen committed:
feat(orga): remove session custom data

when user logs out

igorissen committed:
feat(orga): display language availability banner

on authenticated pages

Commit:
:sparkles: api: update usecase and repository with missing userId property

Commit:
:white_check_mark: api: update acceptance tests to meet new createdByUserId column requirements

Osirisxxl committed:
♻️ api: introduce a unit test context for division certification result

Signed-off-by: Jérémie Picard <jeremie.picard@pix.fr>

machestla committed:
tech(orga): replace contains by testing library in assessment-list integration test

renovate[bot] committed:
[BUMP] Replace dependency eslint-plugin-node with eslint-plugin-n ^14.0.0 (admin)

igorissen committed:
feat: update CODEOWNERS file

add list of bounded countext directories owned by the team accès

igorissen committed:
feat(certif): add Banner::LanguageAvailability component

igorissen committed:
feat(certif): set session data attributes

igorissen committed:
feat(certif): add method updateDataAttribute

to session service

igorissen committed:
feat(certif): handle language availability close action

igorissen committed:
feat(certif): remove session custom data

when user logs out

igorissen committed:
feat(certif): display language availability banner

on authenticated pages

machestla committed:
tech(orga): replace contains by testing library in profile-list integration test

machestla committed:
tech(orga): replace contains by testing library in icon-trigger integration test

aceol committed:
:bug: api: Retrieve the file path from arguments in script

aceol committed:
:bulb: api: Update doc for certification scripts

machestla committed:
tech(orga): replace contains by testing library in organization-credit-info integration test

pix-service-auto-merge committed:
[TECH] Remplacer les notions de `treatment` par `tolerance` (PIX-11679) 

 #8447

pix-service-prod committed:
Updated translations from Phrase

Created locale file nl.json

HEYGUL committed:
fix(1d): add fake id when validating preview challenge

This id is needed only for non-production environment.
See https://github.com/emberjs/data/blob/630ed2678b1259a1b2468280f183c548dd427da1/packages/json-api/src/-private/cache.ts#L877

Co-authored-by: rouxxi <gwenvael.laskar@pix.fr>
Co-authored-by: Aurélie Crouillebois <aurelie.crouillebois@pix.fr>
Co-authored-by: Yannick François <yannick.francois@pix.fr>

machestla committed:
tech(orga): replace contains by testing library in user-logged-menu integration test

pix-service-prod committed:
Updated translations from Phrase

Updated locale file nl.json

pix-service-prod committed:
Updated translations from Phrase

Updated locale file nl.json

pix-service-auto-merge committed:
[BUGFIX] Corrige la validation d'une épreuve en preview en local (PIX-11709). 

 #8470

lego-technix committed:
refactor(api): rename encryption-service→crypto-service

because this service is not only about encrypting, it's also about checking
(checkPassword) and it will have the addition of a decrypt function.

pix-service-auto-merge committed:
[FEATURE] Correctifs traductions NL (PIX-11729) 

 #8469

machestla committed:
tech(orga): replace contains by testing library in organization-participant list integration test

lego-technix committed:
feat(api): add encrypt and decrypt functions to crypto service

nlepage committed:
tech: ignores Phrase generated translations files

pix-service-auto-merge committed:
[FEATURE][MON-PIX] Supprimer le lien politique de protection des données des élèves dans le footer sur le domaine .org (PIX-11650) 

 #8460

pix-service-auto-merge committed:
[FEATURE] Ajout d'une page pour la création de configurations de score et de niveau par compétence (PIX-11639). 

 #8414

lionelB committed:
feat(orga): use import-information api to display import information status

pix-service-auto-merge committed:
[BUGFIX] Permettre de mettre à jour les code pays (cpf) via le script (PIX-11727) 

 #8468

mcampourcy committed:
:card_file_box: api: Add answer id in certification challenges capacities table

lionelB committed:
fix(api): unwrap AggregateImport errors in organizationImport

pix-service-auto-merge committed:
[TECH] :recycle: Déplace `organization-repository.js` vers `src/shared` 

 #8257

mcampourcy committed:
:recycle: api: Add answer id in certification assessment history and in flash algorithm

Commit:
[RELEASE] A minor is being released to 4.119.0.

pix-service-auto-merge committed:
[TECH] Migrer les routes liés au campaign participation dans leurs Bounded Context (PIX-11678) 

 #8461

pix-service-auto-merge committed:
[TECH] Refacto getNextChallengeForCertification (PIX-10870) 

 #8231

aurelie-crouillebois committed:
test(api-1d): fix flaky on assessement-controller_test

pix-service-auto-merge committed:
[TECH] Remplacer les "contains" dans les tests par des méthodes de testingLibrary (PIX-11714) 

 #8476

aurelie-crouillebois committed:
test(api): fix flaky on reconcile-route-admin-post_test

Commit:
:card_file_box: api: add user id to competence-scoring-configurations table

pix-service-auto-merge committed:
[FEATURE] Créer une seeds pour l'import ONDE (PIX-11610) 

 #8409

pix-service-auto-merge committed:
[TECH] Ne pas linter les fichiers de traductions générés par Phrase (PIX-11733) 

 #8471

aceol committed:
:bug: api: Compare NotFoundError from same file

Commit:
:sparkles: api: add user id on competence scoring config creation

pix-service-auto-merge committed:
[BUGFIX] Corriger la gestion d'exception de checkUserBelongsToScoOrganizationAndManagesStudents (PIX-11814) 

 #8478

Commit:
[RELEASE] A minor is being released to 4.120.0.

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [App (.fr)](https://app-pr8482.review.pix.fr)
  * [App (.org)](https://app-pr8482.review.pix.org)
  * [Orga (.fr)](https://orga-pr8482.review.pix.fr)
  * [Orga (.org)](https://orga-pr8482.review.pix.org)
  * [Certif (.fr)](https://certif-pr8482.review.pix.fr)
  * [Certif (.org)](https://certif-pr8482.review.pix.org)
  * [1D](https://1d-pr8482.review.pix.fr)
  * [Admin](https://admin-pr8482.review.pix.fr)
  * [API](https://api-pr8482.review.pix.fr/api/)
  * [Audit Logger](https://pix-audit-logger-review-pr8482.osc-fr1.scalingo.io/api/)

Les variables d'environnement seront accessibles via les liens suivants :
  * [scalingo front](https://dashboard.scalingo.com/apps/osc-fr1/pix-front-review-pr8482/environment)
  * [scalingo api](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr8482/environment)
  * [scalingo audit-logger](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr8482/environment)

Alexandre-Monney committed:
fix(orga): invert 2 column header

Alexandre-Monney committed:
fix(orga/translations): remove one excess whitespace

Alexandre-Monney committed:
feat(orga): add conditional display on participationSharedCount + add missing tests

xav-car committed:
feat(api): add id column in organization-learner-import-formats

pix-service-auto-merge committed:
[FEATURE] Ajouter une colonne id sur la table `organization-learner-import-formats` (PIX-11610) 

 #8484

pix-service-auto-merge committed:
[TECH] Correction de tests échouant aléatoirement (PIX-11751) 

 #8477

RomainPennec committed:
fix(api): replacing emoji looking like buttons (modc-39)

pix-service-auto-merge committed:
[BUGFIX] Inverser les en-têtes de deux colonnes et conditionne un des affichages (PIX-11752) 

 #8483

RomainPennec committed:
fix(api): replace h4 with h3 when necessary

aceol committed:
:white_check_mark: api: Update test to check more complex case

aceol committed:
:sparkles: api: Add getByLabel in complementaryCertificationCourseApi

Alexandre-Monney committed:
fix(api): handle invalid zip error

Alexandre-Monney committed:
fix(api): handle payload too large error

Alexandre-Monney committed:
feat(orga): change link on import information banner

Alexandre-Monney commented on api/tests/prescription/learner-management/integration/infrastructure/repositories/organization-learner-repository_test.js:
```diff
@@ -929,4 +933,142 @@ describe('Integration | Repository | Organization Learner Management | Organizat
+      it('should save the new learner', async function () {
+        const learnerData = new CommonOrganizationLearner({
+          firstName: 'Sacha',
+          lastName: 'Du Bourg Pallette',
+          organizationId,
+          attributes: {
+            firstName: 'Sacha',
+            lastName: 'Du Bourg Pallette',
```
```suggestion
          lastName: 'Du Bourg Palette',
          organizationId,
          attributes: {
            firstName: 'Sacha',
            lastName: 'Du Bourg Palette',
```
On rigole pas avec Pokémon :sob:

yaelle6 replied:
ok la police de pokémon est aux aguets, je vais corriger ça !

pix-service-auto-merge committed:
[BUMP] Replace dependency eslint-plugin-node with eslint-plugin-n ^14.0.0 (admin) 

 #8465

pix-service-auto-merge committed:
[FEATURE] Changer le lien 'Importer' dans la bannière de rentrée vers la nouvelle page d'import (PIX-11822) 

 #8488

renovate[bot] committed:
[BUMP] Replace dependency eslint-plugin-node with eslint-plugin-n ^14.0.0 (mon-pix)

francois2metz committed:
Update eslintrc to use eslint-plugin-n

mirawlin committed:
fix(mon-pix): move aria-live assertive to the whole Grain div

Co-authored-by: Dimitri Lahaye <dimitri.lahaye@pix.fr>

francois2metz committed:
Update eslint-plugin-n

francois2metz committed:
Set files to the package.json

renovate[bot] committed:
[BUMP] Lock file maintenance (api)

pix-service-auto-merge committed:
[BUMP] Replace dependency eslint-plugin-node with eslint-plugin-n ^14.0.0 (mon-pix) 

 #8489

renovate[bot] committed:
[BUMP] Replace dependency eslint-plugin-node with eslint-plugin-n ^14.0.0 (orga)

francois2metz committed:
Move yar plugin configuration to his own file

francois2metz committed:
Run npm run lint:js -- --fix

francois2metz committed:
Enable use of eslint-plugin-n

francois2metz committed:
Update eslint-plugin-n to version 16

pix-service-auto-merge committed:
[FEATURE] utiliser la table `organization-imports` pour afficher les infos du dernier import (Pix-11670) 

 #8474

pix-service-auto-merge committed:
[TECH] Supprimer la configuration des endpoints trouvés automatiquement par OpenID Configuration (PIX-11106) 

 #8458

pix-service-auto-merge committed:
[BUMP] Lock file maintenance (api) 

 #8492

pix-service-auto-merge committed:
[BUMP] Replace dependency eslint-plugin-node with eslint-plugin-n ^16.0.0 (orga) 

 #8493

renovate[bot] committed:
[BUMP] Update dependency eslint-plugin-n to v16 (admin)

pix-service-auto-merge committed:
[FEATURE] Différencier les grains activité des grain leçon (PIX-10857) 

 #8462

pix-service-auto-merge committed:
[FEATURE] Afficher Pix Certif et Pix Orga en anglais lorsque la langue de l'utilisateur n'est pas supportée (PIX-11633) 

 #8467

pix-service-auto-merge committed:
[BUGFIX] Ne pas afficher de grain vide (PIX-11296) 

 #8457

pix-service-auto-merge committed:
[BUMP] Update dependency eslint-plugin-n to v16 (admin) 

 #8497

francois2metz committed:
Use the es6 way to import dotenv

pix-service-auto-merge committed:
[BUGFIX] Éviter les 2 écrans intermédiaires en fin de parcours (PIX-11203). 

 #8395

pix-service-auto-merge committed:
[FEATURE] Ajouter l'info de lien avec un parcours autonome dans le détail d'un profil-cible (PIX-9840). 

 #8363

renovate[bot] committed:
[BUMP] Update dependency ember-flatpickr to v8 (certif)

pix-service-auto-merge committed:
[BUMP] Update dependency ember-flatpickr to v8 (certif) 

 #8499

renovate[bot] committed:
[BUMP] Update dependency ember-flatpickr to v8 (admin)

pix-service-prod committed:
Updated translations from Phrase

Created locale file nl.json

pix-service-auto-merge committed:
[BUMP] Update dependency ember-flatpickr to v8 (admin) 

 #8500

xav-car committed:
fix(api): add id on update-password / generate password usecase

renovate[bot] committed:
[BUMP] Lock file maintenance (audit-logger)

pix-service-auto-merge committed:
[BUMP] Lock file maintenance (audit-logger) 

 #8502

francois2metz committed:
Remove current-lang usage in the API

francois2metz committed:
Remove some remaining usage of current-lang in admin and orga

renovate[bot] committed:
[BUMP] Lock file maintenance (orga)

lego-technix committed:
feat(api): add migration to oidc-providers to add

shouldCloseSession column to oidc-providers table, to store 1 more needed
property.

lego-technix committed:
refactor(api): replace hasLogoutUrl by a generic

shouldCloseSession

lego-technix committed:
refactor(api): replace endSessionUrl by generic

shouldCloseSession to remove the dependency on endSessionUrl which is obtained
through the OpenID Configuration endpoint of the OIDC Provider.

lego-technix committed:
refactor(mon-pix): replace hasLogoutUrl by a generic

shouldCloseSession

lego-technix committed:
refactor(admin): replace hasLogoutUrl by a generic

shouldCloseSession

yaf committed:
:art: use pix ui color

lego-technix committed:
refactor(mon-pix): replace useEndSession by generic

shouldCloseSession to remove the dependency on endSessionUrl which is obtained
through the OpenID Configuration endpoint of the OIDC Provider.

yaf committed:
:art: use pix ui color variables

lego-technix committed:
refactor(api): more logical only call saveIdToken

when shouldCloseSession, which does not make it mandatory to implement
saveIdToken for OIDC Providers having no support for logout.

Co-authored-by: Ismael Gorissen <ismael.gorissen@gmail.com>
Co-authored-by: Emmanuelle Bonnemay <emmanuelle.bonnemay@pix.fr>

Commit:
[RELEASE] A minor is being released to 4.121.0.

pix-service-auto-merge committed:
[TECH] Notifier les membres de l'équipe accès lors d'une modification d'un des domaines (PIX-11836) 

 #8496

renovate[bot] committed:
[BUMP] Update dependency @1024pix/pix-ui to v45 (orga)

Alexandre-Monney committed:
feat(api): add acceptance test on assessment-participations route

francois2metz committed:
Use the released version of eslint-plugin-knex

francois2metz committed:
Switch to released version of eslint-plugin-knex

pix-service-auto-merge committed:
[TECH] Utilise la version moderne ES6 de dotenv 

 #8498

xav-car committed:
fix(orga): update PixSelect declaration

lionelB committed:
feat(api): import-information return an organizationImportDetail

pix-service-auto-merge committed:
[FEATURE] Utiliser les composants Pix UI dans la modale d'ajout de candidat à une session sur Pix Certif (PIX-11588). 

 #8452

xav-car committed:
fix(orga): PixInput* to rigght

Alexandre-Monney committed:
feat(api): add acceptance test on assessment-participation results route

Alexandre-Monney committed:
feat(api): add acceptance test on admin campaign participation route

Alexandre-Monney committed:
refactor(api): move to before each campaignId

xav-car committed:
fix(orga): pixtextarea update params

xav-car committed:
fix(orga): PixCheckbox params

xav-car committed:
fix(orga): PixRadioButton params

xav-car committed:
fix(orga): multiselect params

xav-car committed:
fix(orga): PFASS params

lionelB committed:
feat(orga): show success message when import finish

Alexandre-Monney committed:
refactor(api): remove bookshelf orm from user-orga-settings repository

pix-service-auto-merge committed:
[BUGFIX] Remonter la dernière participation d'un élève à une certification sur le CSV des résultats (PIX-11647) 

 #8464

pix-service-auto-merge committed:
[BUMP] Update dependency @1024pix/stylelint-config to ^5.1.2 (admin) 

 #8387

pix-service-auto-merge committed:
[BUMP] Update dependency @1024pix/stylelint-config to ^5.1.2 (mon-pix) 

 #8401

yaf committed:
:recycle: refactor: use pix ui var instead of direct color value

lionelB committed:
refactor(api): move campaignManagementControlller into campaign bounded context

yaf committed:
:recycle: disable stylelint rules about hex color

pix-service-auto-merge committed:
[BUMP] Update dependency @1024pix/stylelint-config to ^5.1.3 (orga) 

 #8445

renovate[bot] committed:
[BUMP] Update dependency @1024pix/pix-ui to ^44.3.8 (1d)

pix-service-auto-merge committed:
[TECH] Supprime l'usage de current-lang 

 #8503

pix-service-auto-merge committed:
[BUMP] Update dependency @1024pix/pix-ui to ^44.3.8 (1d) 

 #8518

renovate[bot] committed:
[BUMP] Update dependency @1024pix/pix-ui to ^44.3.8 (admin)

pix-service-auto-merge committed:
[BUMP] Lock file maintenance (orga) 

 #8504

renovate[bot] committed:
[BUMP] Update dependency @1024pix/pix-ui to ^44.3.8 (certif)

pix-service-auto-merge committed:
[TECH] Ajoute les tests d'acceptance manquant sur les routes liées aux `campaign-participations` (PIX-11742) 

 #8510

pix-service-auto-merge committed:
[BUMP] Update dependency @1024pix/pix-ui to ^44.3.8 (certif) 

 #8520

pix-service-auto-merge committed:
[TECH] Déplace la configuration du plugin yar dans son propre fichier 

 #8495

pix-service-auto-merge committed:
[TECH] Basculer sur la version publié de eslint-plugin-knex 

 #8508

renovate[bot] committed:
[BUMP] Update dependency @1024pix/pix-ui to ^44.3.8 (orga)

pix-service-auto-merge committed:
[BUMP] Update dependency @1024pix/pix-ui to ^44.3.8 (admin) 

 #8519

renovate[bot] committed:
[BUMP] Update dependency @1024pix/stylelint-config to ^5.1.3 (admin)

pix-service-auto-merge committed:
[BUMP] Update dependency @1024pix/stylelint-config to ^5.1.3 (admin) 

 #8522

pix-service-auto-merge committed:
[BUMP] Update dependency @1024pix/pix-ui to ^44.3.8 (orga) 

 #8521

pix-service-prod committed:
Updated translations from Phrase

Updated locale file nl.json

renovate[bot] committed:
[BUMP] Update dependency @1024pix/stylelint-config to ^5.1.3 (mon-pix)

pix-service-prod committed:
Updated translations from Phrase

Updated locale file nl.json

pix-service-auto-merge committed:
[BUGFIX] Réparer la gestion de l'import en masse de sessions avec certification complémentaire sur Pix Certif (PIX-11818). 

 #8490

pix-service-auto-merge committed:
[TECH] Remplacer l'utilisation de la propriété de configuration endSessionUrl (PIX-11821) 

 #8494

pix-service-auto-merge committed:
[FEATURE] ajout du commentaire automatique en cas d'obtention d'une certification complémentaire de niveau inférieur (PIX-10322) 

 #8466

xav-car committed:
bsr(orga): add missing translation on fr/en

Commit:
[RELEASE] A minor is being released to 4.122.0.

pix-service-prod committed:
Updated translations from Phrase

Updated locale file nl.json

pix-service-auto-merge committed:
[FEATURE] Ajout de l'ID d'answer dans la table `certification-challenge-capacities` (PIX-11688) 

 #8475

xav-car committed:
bsr(orga): fix stages displayed

pix-service-auto-merge committed:
[TECH] Ajout du userId créant une configuration de niveau par compétence (PIX-11721). 

 #8480

pix-service-auto-merge committed:
[FEATURE] Récupérer l'information des langues disponibles pour la certification (pix-11277) 

 #8237

pix-service-prod committed:
Updated translations from Phrase

Updated locale file nl.json

pix-service-auto-merge committed:
[FEATURE] afficher le détail du dernier import réalisé par une organisation (pix-11684) 

 #8509

pix-service-auto-merge committed:
[TECH] deplace `campaignManagementController` vers `src/prescription` (PIX-11861) 

 #8517

lionelB committed:
feat(orga): disable import button when there is a pending import

pix-service-auto-merge committed:
[BUMP] Update dependency @1024pix/pix-ui to v45 (orga) 

 #8507

renovate[bot] committed:
[BUMP] Update Node.js to v20.12.0

pix-service-auto-merge committed:
[BUMP] Update Node.js to v20.12.0 

 #8536

pix-service-auto-merge committed:
[TECH] Ajouter des fonctions de chiffrement et de déchiffrement symétriques pour stocker les Client Secrets (PIX-11883) 

 #8532

Commit:
[RELEASE] A minor is being released to 4.123.0.

pix-service-auto-merge committed:
[BUMP] Update dependency @1024pix/stylelint-config to ^5.1.3 (mon-pix) 

 #8524

renovate[bot] committed:
[BUMP] Update dependency pino-pretty to v11 (api)

pix-service-auto-merge committed:
[BUMP] Update dependency pino-pretty to v11 (api) 

 #8538

renovate[bot] committed:
[BUMP] Update dependency pino-pretty to v11 (audit-logger)

pix-service-auto-merge committed:
[BUMP] Update dependency pino-pretty to v11 (audit-logger) 

 #8539

pix-service-auto-merge committed:
[BUGFIX] Remplacement émojis boutons et correction erreurs de niveau de section  

 #8486

RomainPennec committed:
fix(api): fix tolerance in qroc

xav-car committed:
feat(api): add ModelEntityValidator in order to replace EntityValidationError

xav-car committed:
feat(api): add common organization learner validator from config rules

xav-car committed:
feat(api): Add ComonOrganizationLearnerSet to create  learner to save on bdd

xav-car committed:
feat(api): create common csv parser from specific import configuration

pix-service-auto-merge committed:
[BUGFIX] Màj tolérance d'un QROC du module bien-ecrire-son-adresse-mail 

 #8541

pix-service-auto-merge committed:
[TECH] Mettre à jour ember-data en 4.12.5 sur PixOrga (PIX-11824) 

 #8273

pix-service-auto-merge committed:
[BUGFIX]Corrections de liens menant vers des pages de support NL (PIX-11551) 

 #8528

pix-service-auto-merge committed:
[TECH] Remplacer bookshelf par knex sur le repository user-orga-settings (PIX-11858) 

 #8514

pix-service-auto-merge committed:
[FEATURE] désactive les boutons d'import si un import est en cours (PIX-11681) 

 #8535

pix-service-auto-merge committed:
[FEATURE] Permettre à partir d'un fichier de configuration de parser un CSV (PIX-11614) 

 #8415

pix-service-auto-merge committed:
[TECH] Corriger comportement de screen reader sur l'affichage d'un nouveau grain (PIX-11825). 

 #8491

Commit:
tech(api): save and disable common organization learners
